<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewDistances</class>
 <widget class="QDialog" name="ViewDistances">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="distance_text_browser">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="distance_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QComboBox" name="to_stadium_cbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="from_stadium_cbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="from_stadium_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_stadium_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_distance_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="arrow_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>---&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
